src/framework/scene-registry.js

```diff
@@ -347,6 +358,52 @@ class SceneRegistry {
         });
     }
 
+    /**
+     * Change to a new scene. Calling this function will load the scene data, delete all
+     * entities and graph nodes under `app.root` and load the scene settings and hierarchy.
+     *
+     * @param {SceneRegistryItem | string} sceneItem - The scene item (which can be found with
+     * {@link SceneRegistry#find} or name of the scene.".
+     * @param {ChangeSceneCallback} [callback] - The function to call after loading,
+     * passed (err, entity) where err is null if no errors occurred.
+     * @example
+     * app.scenes.ChangeScene("Scene Name", function (err, entity) {
+     *     if (!err) {
+     *         // success
+     *     } else {
+     *         // error
+     *     }
+     * });
+     */
+    changeScene(sceneItem, callback) {
+        if (!(sceneItem instanceof SceneRegistryItem)) {
```

Comment:
Somewhat shorter without negation:

```suggestion
        if (typeof sceneItem === 'string') {
```